build.gradle.kts

```diff
@@ -54,6 +54,7 @@ subprojects {
 
         implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
         implementation("com.github.jsonld-java:jsonld-java:0.13.0")
+        implementation("com.google.guava:guava:29.0-jre")
```

Comment:
If it is only used in entity-service (as it seems), add the dependency only in the entity-service's build file